manifest.toml

```diff
@@ -44,11 +44,18 @@ ram.runtime = "50M"
         sha256 = "2b5552db642a05edabb21a7187027c3cf3b2e0ec3cfb5921106de99a9453755c"
 
     [resources.system_user]
+    allow_email = true
```

Comment:
```suggestion

```

Reply:
It will be nice if this works, but it didn't for now. By default on instances working, the mails are send from the admin user created with the post-install... 

I can show you a source mail test sended from a paheko and received:

```
Return-Path: <SRS0=vRIb=FY=yuno_domain.tld=paheko__2@domain.tld>
Delivered-To: user_mail@domain.tld
Received: from yuno_domain.tld (yuno_domain.tld [IPv6:XXXX:XXXX:X:XXXX::1])
	by domain.tld (Postfix) with ESMTPS id C679F2C104E
	for <user_mail@domain.tld>; Tue, 10 Oct 2023 17:35:49 +0200 (CEST)
To: user_mail@domain.tld
Subject: test
From: "Administrateur Garradin" <admin_user@yuno_domain.tld>
Message-Id: <65256f547fe14.byuabmveyt@adhesions-ecranvillage.yuno_domain.tld>
List-Unsubscribe: <https://paheko.yuno_domain.tld/?un=GI2J3fHEG7_rkPSYGcreOXq1r1s>
List-Unsubscribe-Post: Unsubscribe=Yes
X-Auto-Response-Suppress: All
Date: Tue, 10 Oct 2023 17:35:48 +0200
Content-Type: multipart/mixed; boundary="==_=_65256f547fd60-b434837cf6"
Content-Transfer-Encoding: 8bit
Mime-Version: 1.0

This is a message in multipart MIME format. 
Your mail client should not be displaying this. 
Consider upgrading your mail client to view this message correctly.

--==_=_65256f547fd60-b434837cf6
Content-Type: multipart/alternative; boundary="alt=_-===_=_65256f547fd60-b434837cf6"


--alt=_-===_=_65256f547fd60-b434837cf6
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable
```
Perhaps is a good idea to make the app send messages, but it needs found how it can works... Or let it like this...

Reply:
I am sorry to persist asking this doesn't works, it was working without these configs

Reply:
What was working ... emails from the app ?

There is "working" and "working". Mails from apps won't be DKIM-signed unless the app sending them properly authenticate on the SMTP server (postfix). This is what is configured with this change. If mails aren't DKIM-signed, sure they may be sent, but there's a good propability that GMail or Microsoft will flag them as spam, or worse silently drop them.